restcomm/configuration/config-scripts/as7-config-scripts/restcomm/autoconfig.d/config-restcomm.sh

```diff
@@ -833,6 +833,22 @@ configConferenceTimeout(){
 	xmlstarlet ed --inplace -u "/restcomm/runtime-settings/conference-timeout" -v "$CONFERENCE_TIMEOUT" $FILE
 }
 
+configSdrService(){
+    if  [ -n "$SDR_SERVICE_CLASS" ]; then
+        echo "Configure Sdr service"
+    	xmlstarlet ed --inplace -d "/restcomm/runtime-settings/sdr-service" \
+                -s "/restcomm/runtime-settings" -t elem  -n sdr-service \
+                -i "/restcomm/runtime-settings/sdr-service" -t attr -n class -v "$SDR_SERVICE_CLASS" \
```

Comment:
is restcomm.xml updated accordingly, or we can assume this will create necessary xml elements?